### ISSUE TITLE
_Replace With a Descriptive Title_

### ISSUE THREAD

**Issue by XQrobby** [deepmodeling/deepmd-kit#2810]
### Summary

I want to know how deepmd processes temperature in neural networks. How to use DeepMD to predict the energy of the same structure at different temperatures?

### DeePMD-kit Version

all

### TensorFlow Version

all

### Details

1.Hello, how does DeepMD predict the energy and force of the same structure at different temperatures? To my knowledge, the loss function of DeepMD does not include a temperature related loss term.

 2.Also, if I use the results of AIMD to train deepmd, should I ensure that the temperature is at 0k?

**Comment by njzjz** [comment]
See the `examples/fparam` directory, where the electronic temperature can be an extra input.

**Comment by njzjz** [comment]
Feel free to reopen if the problem still exists. Thank you.
